firmware/src/widgets/weatherWidget.cpp

```diff
@@ -270,6 +271,12 @@ void WeatherWidget::weatherText(int displayIndex) {
         display.drawString(messageArr[i], centre, y);
         y += 22;
     }
+
+    String lastupdate = model.getLastUpdate();
+    display.setTextSize(1);
+    display.setTextColor(TFT_RED);
+    lastupdate.remove(5);
```

Comment:
It might make sense to add a comment here what part of the string is being removed.